### PR TITLE
Upload "NGS2_WITNESS_Cycle1_difi_exp2.R"

### PR DESCRIPTION
**Pull request by C-Pablo-Diego-Rosell**
Basic analytics for the DIFI data. 

1. Produces basic histograms for the distribution of DIFI scores
2. Produces and saves two correlation matrices 
3. Saves bivariate regressions for each of the two DIFI outcomes.

### CODE REVIEW

File: NGS2_WITNESS_Cycle1_difi_exp2.R [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+lm.overlapScale.frame = list()
+for (i in VarList) {
+  report.lm<-lm.tester("overlapScale", i)
+  report.rows<-length(report.lm[,1])
+  lm.overlapScale.frame[[i]]<-as.data.frame(report.lm[,1:4])[2:report.rows,]
+}
+
+report.lm.overlapScale<-do.call(rbind, lm.overlapScale.frame)
+write.csv(report.lm.overlapScale, file = 'lm_DIFIdistance_exp2.csv', row.names = FALSE, na = '')
```
Comment (mhoover) [inline NGS2_WITNESS_Cycle1_difi_exp2.R]: You're overwriting the results of `report.lm.distanceScale` here. I think you need to change the output file name.

**Review by mhoover — CHANGES_REQUESTED** [review]
Last line needs to get fixed, then I can give it a 👍 .

File: NGS2_WITNESS_Cycle1_difi_exp2.R (hunk 1 above)
Comment (C-Pablo-Diego-Rosell) [reply NGS2_WITNESS_Cycle1_difi_exp2.R]: Done, thanks!

**Review by mhoover — APPROVED** [review]
LGTM, 👍

**Pull request merged**